src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -1761,8 +1761,8 @@ fn resolution_failure(
                         MalformedGenericsVariant::MissingType => {
                             String::from("missing type for generic parameters")
                         }
-                        MalformedGenericsVariant::HasUfcs => {
-                            String::from("UFCS syntax is unsupported")
+                        MalformedGenericsVariant::HasFullyQualifiedSyntax => {
+                            String::from("fully-qualified syntax is unsupported")
                         }
```

Comment:
Can you also link to the issue?
```suggestion
                        MalformedGenericsVariant::HasFullyQualifiedSyntax => {
                            if let Some(span) = sp {
                                diag.span_label("fully-qualified syntax is unsupported");
                            } else {
                                diag.label("fully-qualified syntax is unsupported");
                            }
                            String::from("see https://github.com/rust-lang/rust/issues/74563 for more information")
                        }
```
or maybe just add `diag.note("see ...")` so it's less typing, not sure the ordering will work for that though.

Reply:
```
error: unresolved link to `<Vec<T> as IntoIterator>::iter`
  --> $DIR/intra-link-malformed-generics.rs:15:6
   |
LL | //! [<Vec<T> as IntoIterator>::iter]
   |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ fully-qualified syntax is unsupported
   |
   = note: see https://github.com/rust-lang/rust/issues/74563 for more information
```